unix/src/machine_stats/__init__.py

```diff
@@ -283,6 +283,61 @@ def v2_playbook_on_stats(self, stats):
         self._display.display("", screen_only=True, stderr=True)
 
 
+class MeasurementsResultCallback(ResultCallback):
```

Comment:
I'm not sure if wee need to call parent's (ResultCallback) `__init__` here:

```
def __init__(self, plugins, *args, **kwargs):
    super().__init__(*args, **kwargs)
```

WDYT?

FYI: https://docs.python.org/3/library/functions.html#super

Reply:
Have you checked if `MeasurementsResultCallback` works OK with plugins?

Reply:
> Have you checked if MeasurementsResultCallback works OK with plugins?

It works well with cpu-utilization however, when running with proc-stats, I'm getting the following error with or without enabling `measurement`. Something I should take care of?
![image](https://user-images.githubusercontent.com/30822450/149528826-cf931ce9-f967-4737-bd66-a7cd9b5a947b.png)

Reply:
IIRC, `--cpu-utilization-timeout` also requires `--cpu-utilization` flag to be set.

Reply:
Couldn't find the `--cpu-utilization` flag.
I am getting the same error `too many values to unpack` when running machine-stats with `--process-stats` flag with/without `--cpu-utilization-timeout` and `--measurement` flag.

![image](https://user-images.githubusercontent.com/30822450/149935698-9e101584-580f-4898-8666-b8e31cdac0c6.png)

Reply:
Oh, I see. Indeed, looks like a bug with `--process-stats`. Could you please file a new Issue and assign it to @justinbarclay ?

Reply:
Done. Issue created here: https://github.com/tidalmigrations/machine_stats/issues/149